src/components/issues/Card.tsx

```diff
@@ -107,6 +74,53 @@ const Card: FC<IssueCardProps> = ({ issue }) => {
                     Convert to task
                 </button>
             </div>
+            <p className={styles.card__issue_created__by}>
+                Opened on {created} by
+                <a
+                    href={issue.user.html_url ?? '#'}
+                    target="_blank"
+                    rel="noreferrer"
+                >
+                    {issue.user.login}
+                </a>
+            </p>
+            <div className="card__body">
+                <MarkdownRenderer
+                    className={styles.card__body}
+                    content={issue.body ?? 'No description provided.'}
+                />
+            </div>
+            {issue.assignee && (
+                <p className={styles.card__assignee}>
+                    Assigned to:
+                    <a
+                        href={issue.assignee?.html_url ?? '#'}
+                        target="_blank"
+                        rel="noreferrer"
+                    >
+                        {issue.assignee?.login}
```

Comment:
i cant see test for this as well if the assignee exists or not

Reply:
See this - https://github.com/Real-Dev-Squad/website-status/pull/580/files#diff-d45c985390f469d6629dabf9546686d836be99a3b27bf0a7a41a48f3e32bf8c0R42-R51